.github/workflows/helm-release.yml

```diff
@@ -0,0 +1,52 @@
+name: helm releaser
+
+on:
+  push:
+    branches: [master, main, force_test, release-*, staging, trying]
```

Comment:
In this scenario we want to publish only on `tags`, hence `branches` is not needed here.